pkg/products/monitoring/reconciler.go

```diff
@@ -142,13 +142,33 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 				return phase, err
 			}
 		}
-		return integreatlyv1alpha1.PhaseCompleted, nil
+		_, err = resources.GetNS(ctx, r.Config.GetOperatorNamespace(), serverClient)
+		if !k8serr.IsNotFound(err) {
+			phase, err := resources.RemoveNamespace(ctx, installation, serverClient, r.Config.GetOperatorNamespace())
+			if err != nil || phase != integreatlyv1alpha1.PhaseCompleted {
+				return phase, err
+			}
+		}
+		//if both namespaces are deleted, return complete
+		_, operatorNSErr := resources.GetNS(ctx, r.Config.GetOperatorNamespace(), serverClient)
+		_, nsErr := resources.GetNS(ctx, r.Config.GetNamespace(), serverClient)
+		if k8serr.IsNotFound(operatorNSErr) && k8serr.IsNotFound(nsErr) {
+			return integreatlyv1alpha1.PhaseCompleted, nil
+		}
+		return integreatlyv1alpha1.PhaseInProgress, nil
```

Comment:
Same as I commented on `pkg/products/rhsso/reconciler.go`
IMHO 7 lines above can be replaced with just `return integreatlyv1alpha1.PhaseCompleted, nil`

Reply:
lets just discuss this above.